purchases/src/main/kotlin/com/revenuecat/purchases/CustomerInfoHelper.kt

```diff
@@ -66,6 +71,26 @@ internal class CustomerInfoHelper(
         }
     }
 
+    private fun postPendingPurchasesIfNeededAndFetchCustomerInfo(
```

Comment:
This is the main change of this PR

Reply:
Do we need the `IfNeeded`? At the end of the day this is just syncing pending purchases right and not really checking if it's needed right?